FF1Lib/Dialogs.cs

```diff
@@ -516,6 +521,360 @@ public string FormattedItemName(Item item, bool specialItem = true)
 					case Item.Canoe:
 						formatted = FF1Text.BytesToText(Get(0x2B5D0 + 36, 8)).TrimEnd(' ');
 						break;
+					case Item.None:
+						break;
+					case Item.Lute:

```

Comment:
That's a bit silly, is there a way to have it not do that with switch, or is the best practice here is to use if/else when we need to switch only a few case?

Reply:
This was because it requires all cases to be covered in a switch. I've manually added exceptions to the rule in places we do this, but I think if/else might be better when we only cover a few cases. 

Though, we can decide as a group if this rule is something we want or not